### ISSUE TITLE
Fetal skin edema

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7221]
**Preferred term label:**
Fetal skin edema

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://publications.aap.org/pediatrics/article-abstract/147/3/e2020011601/33244/Congenital-Chylothorax-and-Hydrops-Fetalis-A-Novel?redirectedFrom=fulltext?autologincheck=redirected
(Note -- different definitions at different ages)
Todo -- check numbers/thresholds

**Comment by pnrobinson** [comment]
Label: Fetal skin edema
Definition: A separation of skin in one or more regions of the body or even along the entire length of body wall (to a degree greater than gestational-age related norms) related to pathologically increased fluid accumulation in the subcutaneous tissue. PMID:23696062

Comment:  In first trimester ultrasound examination generalized skin edema is often the first feature of hydrops fetalis.

Parents : Edema HP:0000969 and Abn. fetal morphology

**Comment by pnrobinson** [comment]
added
